Leaf API.postman_collection.json

```diff
@@ -1734,7 +1735,7 @@
 										],
 										"body": {
 											"mode": "raw",
-											"raw": "{\n    \"operationsImageCreation\": false,\n    \"operationsSummaryCreation\": false\n}"
+											"raw": "{\n    \"operationsImageCreation\": false\n}"
```

Comment:
Should we include all the properties here?

Reply:
Done in 1005b0350abaa5483615d613b4390ecc3662bd8c